stencil-library/src/components/refund-form/refund-form-schema.ts

```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+
+export interface RefundFormFields {
+  amount: number | null; // Allowing null for initial state or cleared inputs
+  message: string;
+}
+
+const RefundFormSchema = yup.object().shape({
```

Comment:
We can figure it out in the PR where we get the refund to actually submit, but we should validate that the refund amount is not higher than the payment amount on the payment being refunded (might need to pass that as a prop)